apps/web/src/components/balances/AssetsTable/index.tsx

```diff
@@ -82,6 +91,12 @@ const headCells = [
     width: '20%',
     align: 'right',
   },
+  {
+    id: 'change',
+    label: '(24h change)',
+    width: '20%',
+    align: 'left',
+  },
```

Comment:
The total `width` is above 100%. However, it was like this before? Maybe we should try to reduce these values?

Reply:
I redid the math and split it such that it adds up to 100%.